api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -111,7 +111,7 @@ public void checkFreeTierBillingUsageForUsers(
 
     logger.info(
         String.format(
-            "Workspaces that require updates: %s",
+            "FreeTierBillingUsage: Workspace ids that require updates: %s",
```

Comment:
Thought prefixing it with FreeTierBillingUsage so that its easier for quick log search